ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -1146,6 +1146,18 @@ private boolean isChangeTypeDate(AttributeChange attributeChange, Metacard resul
   }
 
   private List<Result> getMetacardHistory(String id) {
+    Filter resourceFilter = filterBuilder.attribute(Core.ID).is().equalTo().text(id);
+    List<Result> results =
+        ResultIterable.resultIterable(
+                catalogFramework::query, new QueryRequestImpl(new QueryImpl(resourceFilter)))
```

Comment:
❓ If this is adding in support for the history of harvested data then how does this first catalog query work? Wouldn't it need to specify the source of the metacard as well in order to retrieve the metacard?

Reply:
If they are all stored locally, can't we use the local provider id vs. issuing a query to the framework?

Reply:
The harvested data is part of local, what I need to change is to look up the `ext.harvested-from` attribute once I grab the metacard and use that as the source if there is a `ext.harvested-from` attribute value